google/cloud/storage/parallel_upload.h

```diff
@@ -953,16 +962,16 @@ std::vector<std::uintmax_t> ComputeParallelFileUploadSplitPoints(
   };
   // These defaults were obtained by experiments summarized in
   // https://github.com/googleapis/google-cloud-cpp/issues/2951#issuecomment-566237128
-  MaxStreams const kDefaultMaxStreams(64);
-  MinStreamSize const kDefaultMinStreamSize(32 * 1024 * 1024);
+  MaxStreams const k_default_max_streams(64);
```

Comment:
This looks like a style that I"m not familiar with. Could these just be named like `default_max_streams`?

Reply:
Yes.  clang-tidy suggested the `k_` form, but it is fine without it too.